common/src/main/java/org/mcaccess/minecraftaccess/features/point_of_interest/POIBlocks.java

```diff
@@ -110,6 +110,11 @@ public class POIBlocks {
     private final POIGroup<BlockPos>[] groups = new POIGroup[] {
             markedGroup,
             oreGroup,
+            new POIGroup<BlockPos>(
```

Comment:
Please include a comment to indicate what this group is for
```suggestion
            new POIGroup<BlockPos>(// Doors
```